Source        : EntityFramework
Method        : SaveChanges
Date        : 11:27:09 AM
Time        : 12/6/2016
Computer    : udayc7
Error        : An error occurred while updating the entries. See the inner exception for details.
Stack Trace    : at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at TrainingTracker.DAL.DataAccess.LearningPathDal.AddAssignmentSubtopicMapping(Int32 assignmentId, Int32 subtopicId) in d:\Projects\TT\TrainingTracker.DAL\DataAccess\LearningPathDal.cs:line 457
^^-----------------------------------------------------^^
Source        : EntityFramework
Method        : SaveChanges
Date        : 11:34:42 AM
Time        : 12/6/2016
Computer    : udayc7
Error        : An error occurred while updating the entries. See the inner exception for details.
Stack Trace    : at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at TrainingTracker.DAL.DataAccess.LearningPathDal.AddAssignmentSubtopicMapping(Int32 assignmentId, Int32 subtopicId) in d:\Projects\TT\TrainingTracker.DAL\DataAccess\LearningPathDal.cs:line 457
^^-----------------------------------------------------^^
Source        : EntityFramework
Method        : SaveChanges
Date        : 11:37:02 AM
Time        : 12/6/2016
Computer    : udayc7
Error        : An error occurred while updating the entries. See the inner exception for details.
Stack Trace    : at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at TrainingTracker.DAL.DataAccess.LearningPathDal.AddAssignmentSubtopicMapping(Int32 assignmentId, Int32 subtopicId) in d:\Projects\TT\TrainingTracker.DAL\DataAccess\LearningPathDal.cs:line 457
^^-----------------------------------------------------^^
Source        : EntityFramework
Method        : SaveChanges
Date        : 11:46:04 AM
Time        : 12/6/2016
Computer    : udayc7
Error        : An error occurred while updating the entries. See the inner exception for details.
Stack Trace    : at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at TrainingTracker.DAL.DataAccess.LearningPathDal.AddAssignmentSubtopicMapping(Int32 assignmentId, Int32 subtopicId) in d:\Projects\TT\TrainingTracker.DAL\DataAccess\LearningPathDal.cs:line 457
^^-----------------------------------------------------^^
Source        : EntityFramework
Method        : SaveChanges
Date        : 12:04:53 PM
Time        : 12/6/2016
Computer    : udayc7
Error        : An error occurred while updating the entries. See the inner exception for details.
Stack Trace    : at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at TrainingTracker.DAL.DataAccess.LearningPathDal.AddAssignmentSubtopicMapping(Int32 assignmentId, Int32 subtopicId) in d:\Projects\TT\TrainingTracker.DAL\DataAccess\LearningPathDal.cs:line 457
^^-----------------------------------------------------^^
Source        : EntityFramework
Method        : SaveChanges
Date        : 12:05:38 PM
Time        : 12/6/2016
Computer    : udayc7
Error        : An error occurred while updating the entries. See the inner exception for details.
Stack Trace    : at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at TrainingTracker.DAL.DataAccess.LearningPathDal.AddAssignmentSubtopicMapping(Int32 assignmentId, Int32 subtopicId) in d:\Projects\TT\TrainingTracker.DAL\DataAccess\LearningPathDal.cs:line 457
^^-----------------------------------------------------^^
Source        : EntityFramework
Method        : SaveChanges
Date        : 12:09:55 PM
Time        : 12/6/2016
Computer    : udayc7
Error        : An error occurred while updating the entries. See the inner exception for details.
Stack Trace    : at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at TrainingTracker.DAL.DataAccess.LearningPathDal.AddAssignmentSubtopicMapping(Int32 assignmentId, Int32 subtopicId) in d:\Projects\TT\TrainingTracker.DAL\DataAccess\LearningPathDal.cs:line 457
^^-----------------------------------------------------^^
Source        : EntityFramework
Method        : SaveChanges
Date        : 12:13:24 PM
Time        : 12/6/2016
Computer    : udayc7
Error        : An error occurred while updating the entries. See the inner exception for details.
Stack Trace    : at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at TrainingTracker.DAL.DataAccess.LearningPathDal.AddAssignmentSubtopicMapping(Int32 assignmentId, Int32 subtopicId) in d:\Projects\TT\TrainingTracker.DAL\DataAccess\LearningPathDal.cs:line 457
^^-----------------------------------------------------^^
Source        : EntityFramework
Method        : SaveChanges
Date        : 12:15:09 PM
Time        : 12/6/2016
Computer    : udayc7
Error        : An error occurred while updating the entries. See the inner exception for details.
Stack Trace    : at System.Data.Entity.Internal.InternalContext.SaveChanges()
   at System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   at System.Data.Entity.DbContext.SaveChanges()
   at TrainingTracker.DAL.DataAccess.LearningPathDal.AddAssignmentSubtopicMapping(Int32 assignmentId, Int32 subtopicId) in d:\Projects\TT\TrainingTracker.DAL\DataAccess\LearningPathDal.cs:line 457
^^-----------------------------------------------------^^
